src/index.ts

```diff
@@ -58,27 +58,15 @@ export default class BitField {
      * @param i Bit index to set.
      * @param value Value to set the bit to. Defaults to `true`.
      */
-    set(i: number, value = true): void {
+     set(i: number, value = true): void {
```

Comment:
We are using a typed array as the data structure, which doesn't grow when setting a property out of bounds. So this won't work.

Reply:
So the first point maynot be correct but the rest two optimisation are correct

Reply:
```In the get method, the j < this.buffer.length check is unnecessary because the bitwise and operator (&) will return 0 if j is out of bounds.```
Above statement was incorrect